tests/case/case.php

```diff
@@ -1,236 +1,4 @@
 <?php
-DECLARE(ticks=1);
-
-NAMESPACE Vendor\Package;
-
-USE FooClass;
-USE BarClass AS Bar;
-
-INTERFACE iTemplate {}
-
-ABSTRACT CLASS AbstractClass
-{
-    ABSTRACT PROTECTED FUNCTION getValue();
-    ABSTRACT PROTECTED FUNCTION prefixValue($prefix);
-
-    PUBLIC function printOut() {
-        PRINT $this->getValue();
-    }
+FOR ($i = 0; $i < 10; $i++) {
```

Comment:
It seems I accidentally deleted tests, need return /cc @mgrip

Reply:
looks like there are a couple types of comments not getting printed when i add it back. ill get it updated and fixed, then I think we'll be good to merge 👍